### PR TITLE
fix some errors with applio

### PR DESCRIPTION
**Pull request by deeivihh**
- Add [applio.org](https://applio.org) to [how to search models](https://docs.aihub.wtf/essentials/voice-models/#how-to-search-models).
- Fix link at [Local Applio](https://docs.aihub.wtf/rvc/local/applio/).
- Fix cons sections at [Applio Colab](https://docs.aihub.wtf/rvc/cloud/applio-colab/) and [Local Applio](https://docs.aihub.wtf/rvc/local/applio/).

### CODE REVIEW

**Review by Eddycrack864 — COMMENTED** [review]
Seems good for me

File: assets/Essentials/Voice Models.md [hunk 1]
```diff
@@ -95,6 +96,51 @@ If you're curious about the epochs, <u>[learn more here](https://docs.aihub.wtf/
 
 - If it only exists in weights.gg, download the .ZIP & <u>[upload it to HF](https://docs.aihub.wtf/essentials/voice-models/#uploading-to-hugging-face)</u>.  
 
++++ applio.org   
+:::content-center  
+<img src="../searchrvcmodels-img/applio-web-2.png" alt="image" width="350" height="auto">           
+:::  
+
+- This a website where people can find voice models.
+- Not only will you be able to **get models**, but **also guides and use the Applio HF space**, as well as **get access to the Applio API** with **more than 21k models for free**.
```
Comment (SleepyYui) [inline assets/Essentials/Voice Models.md:105]: The English is kinda broken, should have proper grammar and make sense before going online

File: assets/RVC/Cloud/Applio Colab.md [hunk 2]
```diff
@@ -14,7 +14,7 @@ order: 5000
 :::content-center
 ## Introduction ‎
 :::
-- Applio is a <u>[fork](https://docs.aihub.wtf/essentials/whats-rvc/#forks)</u> of <u>[Mangio](https://docs.aihub.wtf/rvc/local/mangio/)</u> developed by the <u>[IA Hispano</u>](https://github.com/IAHispano)</u> team.
+- Applio is a ultimate voice cloning tool developed by the <u>[IA Hispano</u>](https://github.com/IAHispano)</u> team.
```
Comment (SleepyYui) [inline assets/RVC/Cloud/Applio Colab.md:17]: broken English again, also this isn't an ad platform but a documentation so no bias pls ty (refrain from using words like "ultimate" or anything subjective)

File: assets/RVC/Cloud/Applio Colab.md [hunk 3]
```diff
@@ -33,8 +33,8 @@ order: 5000
 - Automatic model upload
 - Has Mangio-Crepe
 - User-friendly UI
+- It is the fastest compared to all other forks
```
Comment (SleepyYui) [inline assets/RVC/Cloud/Applio Colab.md:36]: not objective, there are no independent statistics on that so nope

File: assets/RVC/Local/Applio.md [hunk 4]
```diff
@@ -15,7 +15,7 @@ order: 5000
 :::content-center
 ## Introduction ‎
 :::
-- Applio is a <u>[fork](https://docs.aihub.wtf/essentials/whats-rvc/#forks)</u> of <u>[Mangio](https://docs.aihub.wtf/rvc/local/mangio/)</u> developed by the <u>[IA Hispano</u>](https://github.com/IAHispano)</u> team.
+- Applio is a ultimate voice cloning tool developed by the <u>[IA Hispano</u>](https://github.com/IAHispano)</u> team.
```
Comment (SleepyYui) [inline assets/RVC/Local/Applio.md:18]: once again, "ultimate" is not objective

File: assets/RVC/Local/Applio.md [hunk 5]
```diff
@@ -35,8 +35,8 @@ order: 5000
 - Has Mangio-Crepe
 - User-friendly UI
 - TensorBoard included
+- It is the fastest compared to all other forks
```
Comment (SleepyYui) [inline assets/RVC/Local/Applio.md:38]: _not an objective opinion_

**Review by SleepyYui — CHANGES_REQUESTED** [review]

File: assets/Essentials/Voice Models.md (hunk 1 above)
Comment (deeivihh) [reply assets/Essentials/Voice Models.md:105]: Yes, as you understand, it is not my native language.

File: assets/RVC/Cloud/Applio Colab.md (hunk 3 above)
Comment (deeivihh) [reply assets/RVC/Cloud/Applio Colab.md:36]: I don't understand this, what do you need a graph to show you which Applio is the fastest? Before **it wasn't objective**, now it is.
man

Comment (SleepyYui) [reply assets/RVC/Cloud/Applio Colab.md:36]: I have never said anything on this before, and the graph doesn't show much information therefore I don't think it is good to promote something that can't be proven. Also the docs should display an objective opinion on everything and the suggested edits are clearly biased.

**Comment by aitronz** [comment]
@SleepyYui 
I created another pull request with all the changes revised: https://github.com/AIHubCentral/docs/pull/3

**Pull request closed without merging**